### PR TITLE
WIP: Add notes for build user interfaces by composing css utility classes with tailwind

### PR DESCRIPTION
**Pull request by JacobParis**
Opening a PR for continuous review until I've completed the suite of notes

### CODE REVIEW

**Review by wjohnson85 — COMMENTED** [review]
We would like a readme that includes the course image, a little about the course/instrcutors, and a Table of Contents

**Comment by wjohnson85** [comment]
It's also good to add a link to the video at the top of the md file. and a link to the next video at the bottom
[Go to Next Lesson](https://egghead.io/lessons/lesson-title)

**Comment by JacobParis** [comment]
Uploading course image to Github for free cdn bytes

Build User Interfaces by Composing CSS Utility Classes with Tailwind

**Comment by JacobParis** [comment]
Still missing the Collaborators widget, and I'm not sure how to set that one up

File: build-user-interfaces-by-composing-css-utility-classes-with-tailwind/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Build User Interfaces by Composing CSS Utility Classes with Tailwind
+
+![Build User Interfaces by Composing CSS Utility Classes with Tailwind](https://user-images.githubusercontent.com/5633704/93652072-b9e56d80-f9e1-11ea-9c80-08e960f3aa49.png)
```
Comment (wjohnson85) [inline build-user-interfaces-by-composing-css-utility-classes-with-tailwind/README.md:3]: This course image is gigantic could you center and resize it
**Example:**
`<p align="center"></p>`

**Review by wjohnson85 — COMMENTED** [review]
these notes are really good I like how you added what has changed due to your knowledge of tailwind that is super helpful.

**Pull request merged**